addon/lib/shooter.js

```diff
@@ -150,10 +150,11 @@ const ShotContext = Class({
     this.panelContext = panelContext;
     this._workerActive = false;
     this.watchTab("deactivate", function () {
-      panelContext.hide(this);
+      //panelContext.hide(this);
```

Comment:
can remove (and remove the entire deactive handler, since it no longer calls anything)